lightningd/config

```diff
@@ -0,0 +1,7 @@
+bitcoin-datadir=/etc/bitcoin
+bitcoin-rpcconnect=bitcoind
+
+bind-addr=0.0.0.0:9735
+network=testnet
```

Comment:
this will not work for mainnet

Reply:
changing this so that my script would be run and grab the variable from the env and create it.

Reply:
maybe easier to use the python package jinja2-cli (https://github.com/mattrobenolt/jinja2-cli/blob/d66d6195b2fa4bb3d161e8dbdd2c3e9a8e8e1ecf/samples/environ.jinja2) which can template files using environment variables

Reply:
replacing my script to be the one above.